components/zcash_address/CHANGELOG.md

```diff
@@ -10,10 +10,18 @@ and this library adheres to Rust's notion of
 - `impl serde::{Serialize, Deserialize} for zcash_address::ZcashAddress` behind
   the `serde` feature flag.
 
+### Added
+- `zcash_address::unified`:
+  - `Container::revision`
```

Comment:
This is a change to the `Container` trait, not an addition. `Container` is implementable by anyone outside the crate (as long as they reuse an existing `Container::DataItem` as that is not externally implementable due to the sealed trait bound).